src/components/kytos/tabs/tabs.vue

```diff
@@ -58,15 +58,33 @@ export default {
       content: {},
       notifications: [],
       hiddenPanel: true,
+      lastTab: -1
+    }
+  },
+  watch: {
+    hiddenTabs: function () {
+      /** Watch the hiddenTabs and get all elements with class="tab-nav"
+       * to remove the class "active" when tabs are closed,
+       * and reactive the last used tab
+       **/
+      let tabLinks = $(".tab-nav")
+      if (this.hiddenTabs) {
+        for (let i = 0; i < tabLinks.length; i++) {
+           $(tabLinks[i]).removeClass("active")
+        }
+      }else {
+        if (!$(tabLinks[this.lastTab]).hasClass("active")){
+          $(tabLinks[this.lastTab]).addClass("active")
+        }
+      }
     }
   },
-
   methods: {
     toggleTerminal() {
       this.hiddenTabs = !this.hiddenTabs
     },
     toggleInfoPanelIcon(action) {
-      (action == 'show') ? this.hiddenPanel = false : this.hiddenPanel = true
+      (action === 'show') ? this.hiddenPanel = false : this.hiddenPanel = true
```

Comment:
I did not test it thoroughly, but at least one of the solutions below may have the same meaning:

```suggestion
      this.hiddenPanel = !(action === 'show')
```
Or this:
```suggestion
      this.hiddenPanel = (action !== 'show')
```